src/platform/site-wide/sass/modules/_m-footer.scss

```diff
@@ -102,10 +102,13 @@
       > a {
         display: inline-block;
         color: $color-white;
-        text-decoration: none;
         font-size: 1em;
         font-weight: 400;
         margin: .25em 0;
+
+        &:hover {
+          color: $color-gold !important;
```

Comment:
Normally I'd say let's see if we can just design utility classes, but this makes sense since it's styling within a hover selector 💯